cmd/monitor/main.go

```diff
@@ -0,0 +1,55 @@
+// Service to monitor Adafruit feed uptime and temperatures.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/liftM/fishmon/pkg/adafruitio"
+	"github.com/liftM/fishmon/pkg/ds18b20"
+)
+
+func main() {
+	// Set up command-line flags.
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), `%s starts the feed monitoring service..
+
+Usage of %s:
+`, os.Args[0], os.Args[0])
+		flag.PrintDefaults()
+	}
+	aioUser := flag.String("aio_username", "", "Adafruit.IO username")
+	aioKey := flag.String("aio_key", "", "Adafruit.IO key")
```

Comment:
I don't think we need the key for the API endpoints we're hitting.

Reply:
no, but while authenticating the client we created we'll be hitting an endpoint that requires the key.

Reply:
That endpoint only checks that the key is valid. It may make sense to differentiate between an authenticated vs. unauthenticated client. Since the monitoring binary won't need to be authenticated, I would rather not give it a key (better security hygiene).

Reply:
👍 makes sense